src/main/java/com/spring/boot/project/demo/servise/ReviewsToDbPutter.java

```diff
@@ -0,0 +1,47 @@
+package com.spring.boot.project.demo.servise;
+
+import com.spring.boot.project.demo.model.AmazonUser;
+import com.spring.boot.project.demo.model.Product;
+import com.spring.boot.project.demo.model.Review;
+import com.spring.boot.project.demo.repository.AmazonUserRepository;
+import com.spring.boot.project.demo.repository.ProductRepository;
+import com.spring.boot.project.demo.repository.ReviewRepository;
+import com.spring.boot.project.demo.unit.FileReader;
+import com.spring.boot.project.demo.unit.Parser;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import lombok.AllArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@AllArgsConstructor
+public class ReviewsToDbPutter implements DbPutter {
+    private static final String REVIEW_CSV_FILE = "src/main/resources/csv/Reviews.csv";
+    private final AmazonUserRepository amazonUserRepository;
+    private final ProductRepository productRepository;
+    private final ReviewRepository reviewRepository;
+    private final Parser<Review> parser;
+    private final FileReader fileReader;
+
+    @Override
+    public void putDataToDatabase() {
```

Comment:
try not to use void methods, it's impossible to test them

Reply:
What can we return to test them? If we use boolean, tests wouldn't be objective.

Reply:
yep, you're right, in this case you can leave it with void....